client/Reservation.jsx

```diff
@@ -1,9 +1,52 @@
-import React from 'react';
+  import React, { useState, useEffect } from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
+import styles from './Reservation.css';
 
-const Reservation = () => (
-  <h2>This is a rendering test.</h2>
-);
+const Reservation = () => {
+  const [room, setRoom] = useState([]);
+
+  useEffect(() => {
+    const fetchData = () => {
+      axios.get('/api/room/1')
+        .then((response) => {
+          console.log(response.data[0]);
+          setRoom(response.data[0]);
```

Comment:
Curious, if you switched your server from calling the Model.find(id) method, to using Model.findOne(id) I think you would receive an object rather than an array, but not 100%

Reply:
Excellent suggestion. I'll look into it.